addons/source-python/packages/source-python/menus/radio.py

```diff
@@ -186,11 +187,20 @@ def _format_header(self, player_index, page, slots):
         :param slots: A set to which slots can be added.
         :type slots: :class:`set`
         """
-        # Create the page info string
-        info = '[{0}/{1}]\n'.format(page.index + 1, self.page_count)
 
-        buffer = '{0} {1}'.format(_translate_text(
-            self.title, player_index), info) if self.title else info
+        translated_title = _translate_text(self.title, player_index) if self.title else ''
+        page_info = '[{0}/{1}]'.format(page.index + 1, self.page_count) if self.show_pages else ''
+
+        if translated_title and page_info:
+            buffer = '{0} {1}'.format(translated_title, page_info)
+        elif translated_title:
+            buffer = translated_title
+        elif page_info:
+            buffer = page_info
+        else:
+            buffer = ''
+
+        buffer += '\n'
```

Comment:
Yes, I agree. Changed and tested the code, everything works fine.

Reply:
Done

Reply:
Haven't seen this type of menu. Is this for CS:GO?

Reply:
Ok, I'll take a look later

Reply:
Added support for "ListESCMenu" and "PagedESCMenu"

Reply:
Also, I noticed a bug that the "ESC" menu does not close if you click the "Close" button (bottom right) or the cross (top right). Is it normal?

Reply:
I am planning to add translation for the "Back", "Next" and "Close" buttons. Or make them language-neutral, such as "<", ">" and "X", but this will be in a separate pull request.

Reply:
And why not treat these two buttons as a choice of "0" (Close)?